gulpfile.js

```diff
@@ -104,6 +105,14 @@ var project = {
                 inlinejs: [ uglify() ],
                 inlinecss: [ minifyCss(), 'concat' ]
               }))
+              .pipe(replace({
```

Comment:
This is something we need to do because we run the app in different folders in dev and prod.